week3/src/Components/ActorList.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+
+const gameInfo = [
+	{ 
+    name: '남주혁',
+		src: require("../img/남주혁.jpg"),
+	},
+	{ 
+    name: '안효섭',
+		src: require("../img/안효섭.jpg"),
+  },
+	{ 
+    name: '김선호',
+		src: require("../img/김선호.jpg"),
+	},
+  { 
+    name: '송강',
+		src: require("../img/송강.jpg"),
+	},
+];
+
+const ActorList = () => {
+  const [actors, setActors] = useState([]);
+  const [displays, setDisplays] = useState([]);
+  const [winners, setWinners] = useState([]);
+
+  useEffect(() => {
+    gameInfo.sort(() => Math.random() - 0.5);
+    setActors(gameInfo);
+    setDisplays([gameInfo[0], gameInfo[1]]);
+  }, []);
+
+  const fighterList = actor => () => {
+    if (actors.length <= 2) {
+      if (winners.length === 0) {
+          setDisplays([actor]);
+      } else {
+        let matchWinners = [...winners, actor];
+        setActors(matchWinners);
+        setDisplays([matchWinners[0], matchWinners[1]]);
+        setWinners([]);
+      }
+    } else if (actors.length > 2) {
+      setWinners([...winners, actor]);
+      setDisplays([actors[2], actors[3]]);
+      setActors(actors.slice(2));
+    }
+  };
+
+  return ( 
+    <Container>
+      
+      {displays.map(d => {
+        return (
+          <div className="flex" key={d.name} onClick={fighterList(d)}>
+            <img className="actor-img" src = {d.src}/>
+            <div className="name">{d.name}</div>

```

Comment:
```suggestion
      {displays.map(display => {
        return (
          <div className="flex" key={display.name} onClick={fighterList(display)}>
            <img className="actor-img" src = {display.src}/>
            <div className="name">{display.name}</div>

```

코드의 가독성을 위해서 짧아도 변수명 서술 해주는 건 중요하다고 생각해 !!

Reply:
(죄송합니다 나연님 저희 이번주에 말 놓을 거니까 미리 놓겠습니다!) 
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

고마오😚 참고해서 고쳐볼게!!